build/test.js

```diff
@@ -127,7 +127,7 @@ gulp.task('integration-test', ['serve:nowatch', 'webdriver-update'], runProtract
  * Runs application integration tests. Uses production version of the application.
  */
 gulp.task(
-    'integration-test:prod', ['local-up-cluster', 'serve:prod', 'webdriver-update'],
+    'integration-test:prod', ['serve:prod', 'webdriver-update', 'wait-for-cluster'],
```

Comment:
Sorry, in my previous comment I meant 'integration-test:prod'. But I think you're right, the dev integration tests should also wait for the cluster to spawn. Should I add it?